apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -1,34 +1,35 @@
 import { zodResolver } from "@hookform/resolvers/zod";
```

Comment:
This will be fun solving conflicts with the new booker component  😅 That new component merges the booker + booking form into one page. 

Not much we you can do for now, but it's good to be aware.

See this draft PR from this file down for more context: https://github.com/calcom/cal.com/pull/6792/files#diff-3314f3429ad01dbd0242361878990240275a062a8e0d564b2ec04a856ffae24b